irods/password_obfuscation.py

```diff
@@ -288,6 +288,16 @@ def scramble_v2(s, first_key, second_key):
 
     return scramble(to_scramble, key=hashed_key, scramble_prefix='', block_chaining=True)
 
+def obfuscate_new_password_with_key(new,key):
```

Comment:
Let's use more letters for the parameter names if possible.

`new` and `key` require the reader to look at the implementation to understand what they mean.

Reply:
I quite understand, consider it done.  At the time of the push, I was quite happy just to have it working. Plus it was like 1AM or something... Lol.

Reply:
Done.